### ISSUE TITLE
Двач: клавиатура капча

### ISSUE THREAD

**Issue by ololoe** [TrixiEther/DashchanFork#139]
Почему на капче не показывается цифровая клавиатура? Проблема у макаки?

`  "input": "russian"`

**Comment by TrixiEther** [comment]
Да, это в модуле обрабатывается, надо отсылать numeric или number чтобы корректно обработало

**Comment by ololoe** [comment]
Напиши ему.
